root/vote/VotingIndex.js

```diff
@@ -0,0 +1,381 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+
+type VotingGuideRowProps = {
+  +guideName: string,
+  +mainUrl: string,
+  +showSubscribedArtistsUrl?: boolean,
+};
+
+const VotingGuideRow = ({
+  guideName,
+  mainUrl,
+  showSubscribedArtistsUrl,
+}: VotingGuideRowProps) => {
+  const subscribedArtistsCondition =
+    '&conditions.9.field=artist&conditions.9.operator=subscribed';
+  return (
+    <li>
+      <a href={mainUrl}>
+        {guideName}
+      </a>
+      {showSubscribedArtistsUrl ? (
+        <ul>
+          <li>
+            <a href={mainUrl + subscribedArtistsCondition}>
+              {l('…related to artists in my subscriptions')}
+            </a>
+          </li>
+        </ul>
+      ) : null}
+    </li>
+  );
+};
+
+const VotingIndex = () => (
+  <Layout fullWidth title={l('Voting reports')}>
+    <div id="content">
+      <h1>{l('Voting reports')}</h1>
+
+      <p>
+        {exp.l(
+          `If you’d like to help ensuring the changes made to MusicBrainz data
+           are correct, but do not know where to start, the following reports
+           should be useful. These are pre-defined
+           {edit_search|edit searches}; once you’re comfortable with the edit
+           search you can refine them further or just create your own personal
+           reports for your own use!`,
```

Comment:
Are we sticking with using the "reports" terminology here?

Reply:
Whoops. Nope! Fixed.